README.md

```diff
@@ -62,6 +80,28 @@ Format code:
 make fmt
 ```
```

Comment:
Let's extract "make clean" and "make fmt" into separate section, e.g. "Developer instructions" or "Additional targets" or something like that.

Reply:
Done